onmt/utils/parse.py

```diff
@@ -322,7 +317,7 @@ def validate_train_opts(cls, opt):
 
     @classmethod
     def validate_translate_opts(cls, opt):
-        opt.src_feats = eval(opt.src_feats) if opt.src_feats else {}
+        pass
```

Comment:
Shall we remove this method or do we expect it to be useful in the future?
cc @vince62s

Reply:
I think those two https://github.com/OpenNMT/OpenNMT-py/blob/master/onmt/utils/parse.py#L324-L328 should be merged and we might think a bit about what we want in this, still good to have it, let's make another PR.